tests/unit/bidi/Button.js

```diff
@@ -0,0 +1,133 @@
+define([
+	"intern!object",
+	"intern/chai!assert"
+], function (registerSuite, assert) {
+	var container, html = "<button is='d-button' id='b1' textDir='ltr'>\u05d0\u05d1\u05d2 ABC</button>" +
+		"<button is='d-button' id='b2' textDir='rtl'>ABC \u05d0\u05d1\u05d2</button>" +
+		"<button is='d-button' id='b3' title='ABC \u05d0\u05d1\u05d2' textDir='auto'>\u05d0\u05d1\u05d2 ABC</button>" +
+		"<button is='d-button' id='b4' label='ABC \u05d2\u05d1\u05d0' textDir='auto'>\u05d0\u05d1\u05d2 ABC</button>";
+
+	registerSuite ({
+		name: "deliteful/Button (bidi)",
+		setup: function () {
+			moduleRequire = require.config({
+				context: "module",
+				baseUrl: "../../../",
+				config: {
+					"requirejs-dplugins/has": {
+						"bidi": true
+					}
+				}
+			});
+		},
+		markup: {
+			beforeEach: function () {
+				container = document.createElement("div");
+				document.body.appendChild(container);
+				container.innerHTML = html;
+			},
```

Comment:
This isn't quite right.   You are losing the pointer to the old container `<div>` without first destroying it.